internal/config.go

```diff
@@ -82,6 +82,10 @@ const (
 	StreamSplitterPartitions     = "WALG_STREAM_SPLITTER_PARTITIONS"
 	StreamSplitterBlockSize      = "WALG_STREAM_SPLITTER_BLOCK_SIZE"
 
+	ProfileSamplingRatio = "PROFILE_SAMPLING_RATIO"
+	ProfileMode          = "PROFILE_MODE"
+	ProfilePath          = "PROFILE_PATH"
```

Comment:
@usernamedt Do you think we should add a `WALG_` prefix to these?

Reply:
> @usernamedt Do you think we should add a `WALG_` prefix to these?

I guess that since this is not directly related to the WAL-G functionality, we can leave it as it is.